config/environment_config.yml

```diff
@@ -0,0 +1,32 @@
+defaults: &defaults
+  file_transfer: atos_api/v1/filetransfer/list
+  atos-api: atos_api
+  download-file: atos_api/v1/filetransfer/download/
+local: 
+  <<: *defaults
+  et1_url: http://et1.et.127.0.0.1.nip.io:3100/
```

Comment:
There is a part in the readme that tells you how to setup the system with different domains - so instead of the long et.127.0.0.1.nip.io it can also support custom names if they are defined in the hosts file.  Using the above will mean that can't be used, but no-one uses it anyway so maybe remove that part from the readme ?  The less variations we support the better !